acceleration/docker-compose.yaml

```diff
@@ -0,0 +1,66 @@
+services:
+  web:
+    build: .
+    ports:
+      - "9999:8080"
+    depends_on:
+      - postgres
+      - spiced
+      - spiced-materialized
+    environment:
+      - POSTGRES_CONN=host=postgres port=5432 dbname=postgres user=postgres password=postgres sslmode=disable
+  spiced:
+    build:
+      dockerfile: Dockerfile.spiceai
+    depends_on:
+      - postgres
+    volumes:
+      - ./spicepod.yaml:/root/spicepod.yaml
+    expose:
+      - "50051"
+    deploy:
+      resources:
+        limits:
+          cpus: '2'
+          memory: 2G
+        reservations:
+          cpus: '2'
+          memory: 2G
```

Comment:
```suggestion
```

Unless its required to make the sample work, let's remove this to simplify it

Reply:
this is to make the benchmark more legit

Reply:
Please resolve / address in next iteration.  Closing comment to merge.